src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 12px;
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  text-transform: uppercase;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: stretch;
+  text-align: center;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav {
+  padding: 11px 0;
```

Comment:
It's much better if you add padding at link for accessibility (user who uses a phone will have a bigger space for tap a link)
Hint: make your links with display: block/inline-block
```suggestion
```

Reply:
hi @Viktor-Kost 
i don`t rly understand how to do it, I already have padding on links

Reply:
better make links inline-block. then they will dictate the height for the whole header and won't need to use paddings on the nav(don't forget to reset default margins on `ul`)